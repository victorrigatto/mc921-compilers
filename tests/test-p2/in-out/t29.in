int f() {
  return 0;
}

int main(int argc) {
  return f();
}
