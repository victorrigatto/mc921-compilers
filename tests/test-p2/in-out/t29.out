Program:
    FuncDef:
        Type: int @ 1:1
        Decl: ID(name=f)
            FuncDecl:
                VarDecl:
                    Type: int @ 1:1
        Compound: @ 1:9
            Return: @ 2:3
                Constant: int, 0 @ 2:10
    FuncDef:
        Type: int @ 5:1
        Decl: ID(name=main)
            FuncDecl:
                ParamList:
                    Decl: ID(name=argc)
                        VarDecl:
                            Type: int @ 5:10
                VarDecl:
                    Type: int @ 5:1
        Compound: @ 5:20
            Return: @ 6:3
                FuncCall: @ 6:10
                    ID: f @ 6:10
